core/api/src/graphql/admin/schema.graphql

```diff
@@ -427,7 +426,7 @@ type UsdWallet implements Wallet {
 }
 
 input UserUpdatePhoneInput {
-  accountUuid: ID!
+  accountId: ID!
```

Comment:
will this break admin panel?

Reply:
afaik it's not used in the admin panel. will confirm